.github/pull_request_template.md

```diff
@@ -1,10 +1,22 @@
-**Related Issue(s)**:  ...
+### Proposed Changes
+ <!--- In case of a bug: Describe what caused the issue and how you solved it-->
 
-**Proposed changes**:
-- ...
+### Related Issues
+- fixes #issue-number
+
+### How Did You Test It?
+<!-- unit tests, integration tests, manual verification, instructions for manual tests -->
+
+### Notes for the Reviewer
+<!-- E.g. point out section where the reviewer  -->
+
+### Screenshots (optional)
+<!-- May be added to illustrate the changes -->
 
 ## Pre-flight checklist
-- [ ]  I have read the [contributors guidelines](https://github.com/deepset-ai/haystack/blob/master/CONTRIBUTING.md)
+- [ ] I have read the [contributors guidelines](https://github.com/deepset-ai/haystack/blob/master/CONTRIBUTING.md)
 - [ ] I have [enabled actions on my fork](https://github.com/deepset-ai/haystack/blob/master/CONTRIBUTING.md#forks)
 - [ ] If this is a code change, I added tests or updated existing ones 
```

Comment:
I would keep either this section of the check in the checklist below. Probably an explicit section is better than a checkbox, so I'd remove this in favor of the section above